sdk/sdk-client/src/mocks/mockToken.ts

```diff
@@ -25,5 +25,65 @@ export async function getMockTokenBySymbol(params: {
         name: 'Wrapped Ether',
         decimals: 18,
       })
+    case TokenSymbol.WSTETH:
+      return Token.createFrom({
+        chainInfo: params.chainInfo,
```

Comment:
it shouldn't use params.chainInfo because those addresses only work for Ethereum network, IMO this mock should throw when chainInfo.chainId is different from 1

Reply:
could please add if condition to check for that?